docs/public/configuration/gui.md

```diff
@@ -25,7 +25,7 @@ The supported configuration items are the following.
 * `"enabledLanguages"`: An array of the languages enabled in the GUI, default
   `[ "da", "en", "es", "fi", "fr", "nb", "sv" ]`.
 * `"contactAddress"`: The contact email address displayed in the footer, default
-  `"contact@zonemaster.net"`.
+  `"zonemaster@zonemaster.net"`.
```

Comment:
I fear this will lead to a lot of spam again, and we would have to block this address too. I think we should keep contact@

Reply:
And contact@ is working as intended when I tried. Please try.